modules/howtos/examples/transactions-example.ts

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
+// tag::ts-connect[]
+import {
+  Bucket,
+  Cluster,
+  Collection,
+  connect,
+  GetResult,
+  MutationResult,
+  TransactionDurabilityLevel,
+  Transactions,
+} from 'couchbase'
+import { Couchbase } from 'ottoman'
```

Comment:
this reference to *ottoman* doesn't look like it's used?

(and should be avoided until 4.1. in any case, as I understood it)